client/src/components/Hamburger/index.js

```diff
@@ -45,38 +45,38 @@ const HamburgerInner = styled.span`
   margin-top: -2px;
 
   transition-duration: 0.13s;
-  transition-delay: ${p => (p.isOpen === true ? '0.22s;' : '0.13s')};
+  transition-delay: ${p => (p.isOpen ? '0.22s;' : '0.13s')};
   transition-timing-function: ${p =>
-    p.isOpen === true
+    p.isOpen
       ? 'cubic-bezier(0.215, 0.61, 0.355, 1)'
       : 'cubic-bezier(0.55, 0.055, 0.675, 0.19)'};
   bottom: 0;
 
   transform: ${p =>
-    p.isOpen === true ? 'translate3d(0, -10px, 0) rotate(-45deg)' : ''};
+    p.isOpen ? 'translate3d(0, -10px, 0) rotate(-45deg)' : ''};
 
   &:after {
     ${HamburgerInnerCSS};
-    top: ${p => (p.isOpen === true ? '0' : '-20px')};
+    top: ${p => (p.isOpen ? '0' : '-20px')};
```

Comment:
like the above comment:
```suggestion
    top: ${p => (p.isOpen ? 0 : -20)}px;
```